src/main/java/com/netflix/imflibrary/st0377/header/SoundFieldGroupLabelSubDescriptor.java

```diff
@@ -31,7 +31,7 @@
  * Object model corresponding to SoundFieldGroupLabelSubDescriptor structural metadata set defined in st377-4:2012
  */
 @Immutable
-public final class SoundFieldGroupLabelSubDescriptor extends SubDescriptor
+public class SoundFieldGroupLabelSubDescriptor extends MCALabelSubDescriptor
```

Comment:
In addition to SoundFieldGroupLabelSubDescriptor, AudioChannelLabelSubDescriptor and GroupOfSoundFieldGroupsLabelSubDescriptor can be made to extend MCALabelSubDescriptor, as per  ST 377-4:2021 5.1. 

That may be out of scope for this particular PR, since it's not directly related to S-ADM support, but just wanted to mention it.

Reply:
Correct.
`GroupOfSoundFieldGroupLabelSubDescriptor` could also to be refactored to <cr>`GroupOfSoundfieldGroupsLabelSubDescriptor` in order to match ST 377-4.